metadata-ingestion/src/datahub/sql_parsing/sqlglot_utils.py

```diff
@@ -115,6 +115,8 @@ def _expression_to_string(
     return expression.sql(dialect=get_dialect(platform))
 
 
+PLACEHOLDER_NORMALIZATION_REGEXP = re.compile(r"(%s|\$\d|\?)")
```

Comment:
since this is related to fingerprint, I would call this as PLACEHOLDER_FINGERPRINT_NORMALIZATION or even PLACEHOLDER_BACKWARD_FINGERPRINT_NORMALIZATION